Setup/Methods/Delete.php

```diff
@@ -32,14 +32,14 @@ public function __construct(string $table)
     public function __toString(): string
     {
         return sprintf(
-            "DELETE FROM %s%s",
+            'DELETE FROM %s%s',
             $this->table,
-            [] === $this->conditions
+            ! $this->conditions
```

Comment:
geen negated conditions, dus als je `!` zou willen gebruiken in een ja/nee conditie moet je degene zonder `!` vooraan zetten. (phpstorm kan dit makkelijk voor je omwisselen.)

Echter doen we ook geen `!` op arrays, liever `empty($array)`.